src/Query/Download_Photo.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace UCSC\Blocks\Query;
+
+use UCSC\Blocks\Object_Meta\Photo_Of_The_Week_Meta;
+
+class Download_Photo {
+	
+	public function download_photo_of_the_week_single(): void {
```

Comment:
this is fin to leave in I suppose, but I already pointed out in our meetings that the download should be a link to the image and let the browser handle it natively.